x-pack/plugins/security_solution/public/management/pages/blocklist/constants.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  ExceptionListType,
+  ExceptionListTypeEnum,
+  CreateExceptionListSchema,
+} from '@kbn/securitysolution-io-ts-list-types';
+import {
+  ENDPOINT_BLOCKLISTS_LIST_DESCRIPTION,
+  ENDPOINT_BLOCKLISTS_LIST_ID,
+  ENDPOINT_BLOCKLISTS_LIST_NAME,
+} from '@kbn/securitysolution-list-constants';
+
+export const BLOCKLISTS_LIST_TYPE: ExceptionListType = ExceptionListTypeEnum.ENDPOINT_BLOCKLISTS;
+
+export const BLOCKLISTS_LIST_DEFINITION: CreateExceptionListSchema = {
```

Comment:
There is one (maybe two) more places that need to be updated with the list id. Please add it to: `ALL_ENDPOINT_ARTIFACT_LIST_IDS` const in `x-pack/plugins/security_solution/common/endpoint/service/artifacts/constants.ts:18`.

@dasansol92 I think there is a place on the server side too that needs updating for the "delete" policy use case (fleet server extension) ??

Reply:
Right, it's in x-pack/plugins/security_solution/server/fleet_integration/handlers/remove_policy_from_artifacts.ts:18 but I think we can import there the one in `x-pack/plugins/security_solution/common/endpoint/service/artifacts/constants.ts` and remove the duplication